src/assets/plugins/app.metadata.json

```diff
@@ -12,6 +12,10 @@
     "content-metadata-presets": [
       {
         "id": "app.content.metadata.custom",
+        "disabled": true
+      },
+      {
+        "id": "app.content.metadata.custom2",
```

Comment:
do we need second configuration here?

Reply:
I have made changes so that this change is no longer needed - and to have backward compatibility with already existing extension plugins.